File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/playservices/GooglePlayServicesActivity.java
package com.jdroid.android.sample.ui.google.playservices;

import com.jdroid.android.activity.FragmentContainerActivity;

import androidx.fragment.app.Fragment;

public class GooglePlayServicesActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return GooglePlayServicesFragment.class;
	}

	@Override
	public boolean isGooglePlayServicesVerificationEnabled() {
		return true;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/playservices/GooglePlayServicesActivity.kt
package com.jdroid.android.sample.ui.google.playservices

import androidx.fragment.app.Fragment
import com.jdroid.android.activity.FragmentContainerActivity

class GooglePlayServicesActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return GooglePlayServicesFragment::class.java
    }

    override fun isGooglePlayServicesVerificationEnabled(): Boolean {
        return true
    }
}